omnigibson/systems/macro_particle_system.py

```diff
@@ -418,11 +475,21 @@ def generate_group_particles(
 
         # Standardize orientations and links
         obj = cls._group_objects[group]
+        is_cloth = obj.prim_type == PrimType.CLOTH
```

Comment:
nit: I know this is cheap, but is it possible to compute this only once and cache the result (e.g. when the group is first created?)? Right now this line has been repeated 7 times in the file.

Reply:
there's not a clean way to store it, without adding additional redundant overhead. For now I created an explicit API so at least the implementation is unified across all usages